finality-aleph/src/abft/legacy.rs

```diff
@@ -33,6 +33,12 @@ where
     C: HeaderBackend<B> + Send + 'static,
     ADN: Network<LegacyNetworkData<B>> + 'static,
 {
+    // Remove this check once we implement one on the AlephBFT side (A0-2583).
+    // Checks that the total time of a session is at least 7 days.
+    sanity_check_round_delays(
+        config.max_round,
```

Comment:
Does `config.max_round` denote the highest possible round, or the number of rounds? If the first one then given that rounds start from `0`, there's one extra round not counted in `sanity_check_round_delays_inner`.
(this probably doesn't affect this check's outcome for the test cases anyway)

Reply:
Good point. So actually I have now checked in AlephBFT and it seems this is the highest possible round. However in practice, during the last crash they reached only round 4999, which probably means that something is crashing and the last unit is not created. It's not a big deal, because we should never allow this high round to be reached anyway. 

Anyway, I have changed the loop from `0..max_round` to `0..=max_round` in the sanity_check code. This way it is consistent with the intention of AlephBFT.